app/views/users/_comments_locked_cta.html.erb

```diff
@@ -0,0 +1,27 @@
+<div id="comments-locked-cta" class="crayons-card mt-2 m:mt-0 mb-2 m:p-8 p-4">
```

Comment:
Similar to [this other comment](https://github.com/forem/forem/pull/18681#discussion_r1013468186) from another separate PR, we need to initialize the tracking of the these clicks with `trackCreateAccountClicks('comments-locked-cta');`. What this function call does is "find any elements within this `id` that have `data-tracking-id` and `data-tracking-source` and ensure clicks on them are tracked".

That line can be added at the very end of [base.jsx here](https://github.com/forem/forem/blob/main/app/javascript/packs/base.jsx#L129)

Reply:
@fdocr Thanks for connecting the dots and explaining it. I have applied the suggestion.